src/pages/Home/components/Map/index.tsx

```diff
@@ -0,0 +1,109 @@
+import { useEffect, useRef } from 'react';
+import useGeolocation from 'utils/hooks/useGeolocation';
+import marker from 'pages/Home/static/marker';
+import styles from './Map.module.scss';
+
+interface Props {
+  latitude: number;
+  longitude: number;
+  placeName: string;
+}
+const options = {
+  maximumAge: 1000,
+};
+export default function Map(): JSX.Element {
+  const { location } = useGeolocation(options);
+  const mapRef = useRef<HTMLElement | null | any>(null);
+  const markerRef = useRef<any | null>(null);
+  const selectedMarker = useRef<any | null>(null);
```

Comment:
```suggestion
  const mapRef = useRef<naver.maps.Map | null>(null);
  const markerRef = useRef<naver.maps.Marker | null>(null);
  const selectedMarker = useRef<naver.maps.Marker | null>(null);
```
타이핑 가능하면 any는 지양해주세요.

Reply:
수정했습니다. [4afa320](https://github.com/BCSDLab/JJBAKSA_FRONT_END/pull/50/commits/4afa320d9e2434142e8aab5b21585abaecdd83b4)